src/topology/builder.rs

```diff
@@ -424,14 +424,12 @@ pub async fn build_pieces(
         let typetag = sink.inner.get_component_name();
         let input_type = sink.inner.input().data_type();
 
-        if config.schema.validation {
```

Comment:
Removing this if statement would be part of https://github.com/vectordotdev/vector/issues/16683, not https://github.com/vectordotdev/vector/issues/16534. It would be fine to include here if you are also including the other parts, such as checking if the sink contains an event with log namespacing enabled before actually doing the checks, but without that this is forcing sinks validation for _all_ sinks which we don't want